include/pagmo/algorithms/mbh.hpp

```diff
@@ -393,6 +393,16 @@ class mbh
     {
         return m_algorithm;
     }
+    /// Setter for the inner algorithm.
```

Comment:
Isn't this superfluous? We have the getter above that already returns a non-const reference.